llvm/lib/SYCLLowerIR/SYCLUtils.cpp

```diff
@@ -10,10 +9,69 @@
 //===----------------------------------------------------------------------===//
 #include "llvm/SYCLLowerIR/SYCLUtils.h"
 #include "llvm/IR/Instructions.h"
+#include "llvm/SYCLLowerIR/ESIMD/ESIMDUtils.h"
 
 namespace llvm {
 namespace sycl {
 namespace utils {
+
+using namespace llvm::esimd;
+bool isAddressArgumentInvokeSIMD(const CallInst *CI) {
+  constexpr char INVOKE_SIMD_PREF[] = "_Z33__regcall3____builtin_invoke_simd";
```

Comment:
please factor out. There is another user:
`llvm/lib/SYCLLowerIR/LowerInvokeSimd.cpp:constexpr char INVOKE_SIMD_PREF[] = "_Z33__regcall3____builtin_invoke_simd";`